### PR TITLE
Ft asset sub category #155322115

### PR DESCRIPTION
**Pull request by nelmandela**
### What does this PR do?
  •Creates asset sub category

### Description of work done
  •Add asset sub category model in **model.py** and tests in **test_asset_sub_category_model.py**

### How can this be manually tested?
  •*python manage.py test*

### Relevant pivotal tracker stories
  #155322115

### CODE REVIEW

**Comment by egichuri** [comment]
@nelmandela please fix the failures on CI. also make sure you have the pre-commit hook set up.

**Comment by nelmandela** [comment]
I've installed the pre-commit hooks.Solved the failures but upon pulling latest develop its failing on my local,working on it.

**Comment by nelmandela** [comment]
@egichuri the tests are passing but coverage is reducing, kindly check it out & advice the way forward

**Comment by egichuri** [comment]
@nelmandela [this](https://stackoverflow.com/a/29077523) should help

**Comment by nelmandela** [comment]
Tried it with @jmnyarega to no avail.can we have a sync on this?

File: api/tests/test_asset_sub_category_model.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+    def test_can_delete_a_sub_category(self):
+        self.assertEqual(AssetSubCategory.objects.count(), 1)
+        self.subcategory.delete()
+        self.assertEqual(AssetSubCategory.objects.count(), 0)
+
+    def test_asset_subcategory_model_string_representation(self):
+        sub_category = AssetSubCategory.objects.get(id=1)
+        sub_category_name = "{0}".format(sub_category.sub_category_name)
+        self.assertEquals(sub_category_name, 'Computer Accessories')
```
Comment (egichuri) [inline api/tests/test_asset_sub_category_model.py]: @nelmandela this should be tested directly on the object, not the `sub_category_name ` field.
`self.assertEquals(str(sub_category), 'Computer Accessories')`

File: api/tests/test_asset_sub_category_model.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        )
+
+    def test_can_delete_a_sub_category(self):
+        self.assertEqual(AssetSubCategory.objects.count(), 1)
+        self.subcategory.delete()
+        self.assertEqual(AssetSubCategory.objects.count(), 0)
+
+    def test_asset_subcategory_model_string_representation(self):
+        # sub_category = AssetSubCategory.objects.get(id=1)
+        # sub_category_name = "{0}".format(sub_category.sub_category_name)
```
Comment (jmnyarega) [inline api/tests/test_asset_sub_category_model.py]: Please avoid pushing commented code

**Pull request merged**